guides/v2.3/inventory/inventory-api-reference.md

```diff
@@ -0,0 +1,133 @@
+
+## Inventory Configuration API
+
+The following services reside under the [`Magento/InventoryConfigurationApi/Api`](https://github.com/magento-engcom/msi/tree/2.3-develop/app/code/Magento/InventoryConfigurationApi/Api) namespace, replacing deprecated services and APIs.
+
+Service | Description | Deprecated API
+--- | --- | ---
+[`GetStockItemConfigurationInterface`](https://github.com/magento-engcom/msi/blob/2.3-develop/app/code/Magento/InventoryConfigurationApi/Api/GetStockItemConfigurationInterface.php)  |  Service to retrieve stock item configuration by SKU and stock id |  Catalog inventory configuration stored in [`StockItem`](https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/CatalogInventory/Api/Data/StockItemInterface.php) entity
+[`SaveStockItemConfigurationInterface`](https://github.com/magento-engcom/msi/blob/2.3-develop/app/code/Magento/InventoryConfigurationApi/Api/SaveStockItemConfigurationInterface.php)  | Service to save stock item configuration  |  Stock configuration saved using [`StockItemRepositoryInterface::save`](https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/CatalogInventory/Api/StockItemRepositoryInterface.php#L25)
+
+## Inventory Source Selection Algorithm (SSA) API
+
+The following services reside under the [`Magento/InventorySourceSelectionApi/Api`](https://github.com/magento-engcom/msi/tree/2.3-develop/app/code/Magento/InventorySourceSelectionApi/Api) namespace.
+
+Service | Description
+--- | ---
+[`GetDefaultSourceSelectionAlgorithmCodeInterface`]  |  Service to get the default Source Selection Algorithm (SSA) code
+[`GetSourceSelectionAlgorithmListInterface`]  |  Service to get the entire Source Selection Algorithm (SSA) list
+[`SourceSelectionServiceInterface`]  |  Service to run a specified Source Selection Algorithm (SSA) based on provided Inventory Request object
+
+## Inventory Export Aggregated Stock Data API
+
+The following services reside under the [`Magento/InventoryExportStockApi/Api`](https://github.com/magento-engcom/msi/tree/2.3-develop/app/code/Magento/InventoryExportStockApi/Api) namespace.
+
+Service | Description
+--- | ---
+[`ExportStockIndexDataInterface`](https://github.com/magento-engcom/msi/blob/2.3-develop/app/code/Magento/InventoryExportStockApi/Api/ExportStockIndexDataInterface.php)  |  Service to return salable quantities for a provided sales channel not including placed reservations for completed orders. Less precise with high performance and speed.
+[`ExportStockSalableQtyInterface`](https://github.com/magento-engcom/msi/blob/2.3-develop/app/code/Magento/InventoryExportStockApi/Api/ExportStockSalableQtyInterface.php)  |  Service to return salable quantities for a provided sales channel including placed reservations (for completed orders). Preceise with reduced performance and slow speed.
```

Comment:
Typo: Preceise  Can't figure out what it should be.

Reply:
Revised these two, thanks!